faster_whisper/transcribe.py

```diff
@@ -1088,3 +1641,12 @@ def merge_punctuations(alignment: List[dict], prepended: str, appended: str) ->
         else:
             i = j
         j += 1
+
+def find_numeral_symbol_tokens(tokenizer):
```

Comment:
This function was added as a means to suppress numeric symbols and currency symbols. But this will be used only if we are using a separate ASR model for alignment purposes.  Actually, I can avoid it for whisper transcription. Thanks for pointing this out.